agent-plane/agent-plane-protocol/src/main/java/org/eclipse/tractusx/agents/edc/service/DataspaceSynchronizer.java

```diff
@@ -19,28 +19,34 @@
 import jakarta.json.Json;
 import jakarta.json.JsonValue;
 import org.apache.jena.atlas.lib.Sink;
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.NodeFactory;
 import org.apache.jena.riot.Lang;
 import org.apache.jena.riot.RDFParser;
 import org.apache.jena.riot.lang.StreamRDFCounting;
 import org.apache.jena.riot.system.ErrorHandler;
 import org.apache.jena.riot.system.ErrorHandlerFactory;
 import org.apache.jena.riot.system.StreamRDF;
 import org.apache.jena.riot.system.StreamRDFLib;
+import org.apache.jena.sparql.core.Quad;
+import org.eclipse.edc.spi.monitor.Monitor;
+import org.eclipse.edc.spi.query.Criterion;
+import org.eclipse.edc.spi.query.QuerySpec;
 import org.eclipse.tractusx.agents.edc.AgentConfig;
 import org.eclipse.tractusx.agents.edc.MonitorWrapper;
 import org.eclipse.tractusx.agents.edc.jsonld.JsonLd;
 import org.eclipse.tractusx.agents.edc.model.DcatCatalog;
 import org.eclipse.tractusx.agents.edc.model.DcatDataset;
-import org.eclipse.tractusx.agents.edc.rdf.RDFStore;
-import org.apache.jena.graph.Node;
-import org.apache.jena.graph.NodeFactory;
-import org.apache.jena.sparql.core.Quad;
-import org.eclipse.edc.spi.monitor.Monitor;
-import org.eclipse.edc.spi.query.Criterion;
-import org.eclipse.edc.spi.query.QuerySpec;
+import org.eclipse.tractusx.agents.edc.rdf.RdfStore;
```

Comment:
What is the purpose of renaming RDFStore to RdfStore when e.g. org.apache.jena.riot.system.StreamRDFLib has still an upper case RDF in it?

Reply:
checkstyle only checks tractus-x code ... apache jena can do whatever they want. And in tractus-x, multiple capital letters are not allowed, we should use Camel case.